Models/SoundEffects/SoundEffect.cs

```diff
@@ -0,0 +1,75 @@
+﻿using static MMRando.Models.SoundEffects.SoundEffectAttributes;
+using static MMRando.Models.SoundEffects.SoundEffectTag;
+
+namespace MMRando.Models.SoundEffects
+{
+    /// <summary>
+    /// Sound effects used throughout the game. 
+    /// The value of a sound effect corresponds to its Id
+    /// 
+    /// Extensive overview of sound ids can be found in this google spreadsheet:
+    /// https://docs.google.com/spreadsheets/d/1YVJ7GdzNZUese6H8d40lzpxp8ZfWlt75c-lzBzKngmo/edit#gid=1343879920
+    /// </summary>
+
+    public enum SoundEffect
+    {
+        // Environment
+
+        [Replacable(0x00EABA46,0x00EABC3A, 0x00EACACA), Tags(Long)]
+        CuccoMorning = 0x2013,
+        [Replacable(0x00EABC4A, 0x00EACABE), Tags(Long)]
+        WolfHowlEvening = 0x20AE, 
+        [Replacable(0x00DFC776), Tags(Short, LowHpBeep), ReplacableByTags(Short)]
+        DogBark = 0x20D8,
+        [Tags(Long)]
+        CowMoo = 0x20DF,
+        [Replacable(0x00DFC7B6), Tags(Short, LowHpBeep), ReplacableByTags(Short)]
+        DogBarkAngry = 0x2110,
+
+        // System
+
+        [Replacable(0x00B97E2A), Tags(Short), ReplacableByTags(LowHpBeep)]
```

Comment:
Shouldn't this also have tag `LowHpBeep` or is the intention to now allow it to replace itself by chance? I think it should sometimes be vanilla.